packages/main/src/Badge.js

```diff
@@ -114,11 +116,8 @@ class Badge extends UI5Element {
 	}
 
 	onBeforeRendering() {
-		if (this.hasIcon) {
-			this.setAttribute("__has-icon", "");
-		} else {
-			this.removeAttribute("__has-icon");
-		}
+		this.toggleAttribute("__has-icon", this.hasIcon);
```

Comment:
I think toggleAttribute is not supported in IE. And we should keep IE working until the end of the year.

Reply:
Removed it.